hs_core/templates/400.html

```diff
@@ -292,7 +292,5 @@ <h1 class="animated slideInLeft"><span>{% block title %}{% endblock %}</span></h
         </div>
     </div>
 
-    {% include "includes/footer_scripts.html" %}
-
```

Comment:
This referenced file does not exist anymore. I'm assuming it was deleted at some point, but this reference was forgotten. Can someone confirm?